internal/io/spinner.go

```diff
@@ -0,0 +1,21 @@
+package io
+
+import (
+	"os"
+
+	"github.com/charmbracelet/huh/spinner"
+	"github.com/mattn/go-isatty"
+)
+
+func SpinWhile(name string, action func()) error {
+	if isatty.IsTerminal(os.Stdout.Fd()) {
```

Comment:
This should have been:


```suggestion
	if !isatty.IsTerminal(os.Stdout.Fd()) {
```